index.bs

```diff
@@ -742,6 +743,57 @@ self.onmessage = async function(e) {
 }
 </pre>
 
+# Implementation advice # {#implementation-advice}
+
+This section is informative.
+
+## Use with multiple consumers
+
+There are use cases where the programmer may desire that a single stream of frames
+is consumed by multiple consumers.
+
+Examples include the case where the result of a background blurring function should
+be both displayed in a self-view and encoded using a {{VideoEncoder}}.
+
+However, the standard tee() operation is problematic
+in this context:
+
+*   It defeats the backpressure mechanism that guards against excessive queueing
+*   It creates multiple links to the same buffers, meaning that the question of which
+    consumer gets to destroy() the buffer is a difficult one to address
+
+Therefore, the use of tee() with Streams containing media should only be done when
+fully understanding the implications. Instead, custom elements for splitting streams
+more appropriate to the use case should be used.
+
+For cases where both consumers are consuming unprocessed frames, and synchronization
+is not desired, instantianting multiple {{MediaStreamTrackProcessor}} objects is a robust solution.
+
+For cases where both consumers intend to convert the result of a processing step into a
+{{MediaStreamTrack}}
+using a {{MediaStreamTrackGenerator}}, attaching the resulting {{MediaStreamTrack}} to
+multiple sinks may be the most appropriate mechanism.
+
+When the stream is the output of some processing, and both branches want to do further processing,
+the correct way of splitting the stream depends on context:
```

Comment:
There is no need to split the stream in two streams as tee does.
For the webcodecs/MSTG example it suffices to clone the processed frame and send one VideoFrame to each consumer, no need to create other readable streams. We should mention this as the preferred solution for this use case.

The case for splitting a stream into two streams is the one I fail to see a use case for. I'm OK with explaining this as an alternative to tee(), but I still don't see any use case that requires tee().

Reply:
I see your point - I see creating a stream as one possible implementation of "send one VideoFrame to each consumer"; if the downstream API expects a stream, then a stream must be created; if it doesn't need one, we shouldn't need to create one.